Representable/representable
# Added Entry Page Front End Changes

Note: Related to  #99, #83, and #73 

- Updates the front-end for the entry page with collapsible sections.
- Adds "Next" and "Previous" buttons to sections.
- Adds "Edit Section"/"Hide Section" button that opens/closes the section.
- Adds JS logic in geo.js to handle section states.
- Adds JS logic for basic error handling and map resizing.

Main steps to test:
- Open Entry Page.
- Write Zipcode. Press Next.
- Press Next a few times until the very end.
- Attempt Hiding/Editing sections multiple times, especially section 1 and section 3.
- When pressing "Previous" and "Next" check that state of "Edit Section" and "Hide Section" button changes accordingly.

To do:
- Integrate with Somya's Branch.
- Add help buttons.
- Add more descriptive errors.
- Add the ability to save state on page refresh. 
<img width="1002" alt="Screen Shot 2020-05-20 at 15 23 37" src="https://user-images.githubusercontent.com/8714338/82488446-e4640000-9aad-11ea-88d6-a28123016c48.png">
<img width="980" alt="Screen Shot 2020-05-20 at 15 23 07" src="https://user-images.githubusercontent.com/8714338/82488392-d1e9c680-9aad-11ea-8d48-4da583b7c669.png">
<img width="926" alt="Screen Shot 2020-05-20 at 15 23 24" src="https://user-images.githubusercontent.com/8714338/82488416-dc0bc500-9aad-11ea-80bb-623cb2c7ce06.png">
